### PR TITLE
rte overflow fix

### PR DESCRIPTION
**Pull request by benbaler**
closes https://github.com/hibobio/hibob/issues/32386

### CODE REVIEW

File: projects/ui-framework/bob-rte/src/rte/rte.component.scss [hunk 1]
```diff
@@ -15,7 +15,6 @@
 
 .bfe-wrap {
   position: relative;
-  z-index: 0;
```
Comment (omrihecht) [inline projects/ui-framework/bob-rte/src/rte/rte.component.scss]: i'm not sure if this is the place...
this is the css of every form-element in bob
@galloween wdyt?

Comment (benbaler) [reply projects/ui-framework/bob-rte/src/rte/rte.component.scss]: @omrihecht its only for rte

Comment (galloween) [reply projects/ui-framework/bob-rte/src/rte/rte.component.scss]: @omrihecht @benbaler 
no, this is scoped to RTE, but
this change will re-introduce this bug: https://github.com/hibobio/hibob/issues/32155
different solution is needed, I will take a look

**Comment by galloween** [comment]
closing this, fix in this PR: https://github.com/hibobio/bob-style/pull/1338 (problem is not rte, but tabs)

**Pull request closed without merging**